core/lib/libtomcrypt/src/tee_ltc_provider.c

```diff
@@ -956,7 +880,7 @@ TEE_Result crypto_acipher_rsaes_encrypt(uint32_t algo,
 
 	ltc_res = rsa_encrypt_key_ex(src, src_len, dst,
 				     (unsigned long *)(dst_len), label,
-				     label_len, &prng->state, prng->index,
+				     label_len, NULL, find_prng("prng_mpa"),
```

Comment:
Should we care about the overhead of looking up "prng_mpa" each time?

Reply:
I don't think it matters much here since the other operations are quite expensive.

Reply:
Probably, yes.